GenderPayGap.WebUI/Views/ManageOrganisations/ManageOrganisations.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@using GenderPayGap.Database
+@model GenderPayGap.WebUI.Models.Organisation.ManageOrganisationsViewModel
+
+@{
+    ViewBag.Title = "Manage your organisations - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        await Html.RenderPartialAsync("../Components/Navigation/AccountTabs");
+    }
+    
+    <div class="govuk-!-margin-top-3">
+        @{
+            await Html.RenderPartialAsync("../Components/CovidMessage");
+        }
+    </div>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Select an organisation
+        </h1>
+        
+        <p class="govuk-body">
+            Use this page to access your registered organisations or to register a new organisation.
+        </p>
+        
+        <p class="govuk-body">
+            Once you have selected an organisation you can:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>Enter gender pay gap data and save it as a draft for publication at a later date</li>
+            <li>Publish your data on the <a class="govuk-link" href="@Url.Action("Index", "Viewing")">gender pay gap service</a></li>
+            <li>Declare whether your organisation is required to report for each reporting year</li>
+        </ul>
+        
+        <table class="govuk-table">
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th scope="col" class="govuk-table__header">Organisation name</th>
+                    <th scope="col" class="govuk-table__header">Registration status</th>
+                </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+                @foreach (UserOrganisation userOrganisation in Model.UserOrganisations)
+                {
+                    string statusText;
+                    string linkAction;
+                    string encryptedOrganisationId = Encryption.EncryptQuerystring(userOrganisation.OrganisationId.ToString());
+                    if (userOrganisation.PINSentDate != null && userOrganisation.PINConfirmedDate == null)
+                    {
+                        statusText = "Awaiting activation PIN";
+                        linkAction = Url.Action("ActivateOrganisation", "Organisation", new {id = encryptedOrganisationId});
+                    } else if (userOrganisation.PINSentDate == null && userOrganisation.PINConfirmedDate == null)
+                    {
+                        statusText = "Awaiting registration approval";
+                        linkAction = null;
+                    }
+                    else
+                    {
+                        statusText = "Registration complete";
+                        linkAction = Url.Action("ManageOrganisationGet", "ManageOrganisations", new {id = encryptedOrganisationId});
+                    }

```

Comment:
I wasn't sure if this chunk of C# was better than the old version (in `Views/Organisation/ManageOrganisations.cshtml`) but it separated it out a bit from lots of processing alongside lots of html?